Pod/Classes/iOS/UITapGestureRecognizer+RxGesture.swift

```diff
@@ -43,10 +43,10 @@ public struct TapGestureRecognizerFactory: GestureRecognizerFactory {
      - parameter configuration: A closure that allows to fully configure the gesture recognizer
      */
     public init(
-        numberOfTouchesRequired: Int = Defaults.numberOfTouchesRequired,
-        numberOfTapsRequired: Int = Defaults.numberOfTapsRequired,
-        configuration: ((UITapGestureRecognizer, RxGestureRecognizerDelegate) -> Void)? = Defaults.configuration
-        ){
+        numberOfTouchesRequired: Int = UITapGestureRecognizerDefaults.numberOfTouchesRequired,
+        numberOfTapsRequired: Int = UITapGestureRecognizerDefaults.numberOfTapsRequired,
+        configuration: ((UITapGestureRecognizer, RxGestureRecognizerDelegate) -> Void)? = UITapGestureRecognizerDefaults.configuration
```

Comment:
Why did you replace `Default` by the whole name? The typealias trick was quite good.

Reply:
I'll change this. I did not realise this part honestly.